sdv/evaluation/single_table.py

```diff
@@ -0,0 +1,91 @@
+"""Methods to compare the real and synthetic data for single-table."""
+
+import sdmetrics.reports.utils as report
+from sdmetrics.reports.single_table.diagnostic_report import DiagnosticReport
+from sdmetrics.reports.single_table.quality_report import QualityReport
+
+
+def evaluate_quality(real_data, synthetic_data, metadata, verbose=True):
+    """Evaluate the quality of the synthetic data.
+
+    Args:
+        real_data (pd.DataFrame):
+            The table containing the real data.
+        synthetic_data (pd.DataFrame):
+            The table containing the synthetic data.
+        metadata (SingleTableMetadata):
+            The metadata object describing the real/synthetic data.
+        verbose (bool):
+            Whether or not to print report summary and progress.
+            Defaults to True.
+
+    Returns:
+        float
+            The overall quality score.
+    """
+    quality_report = QualityReport()
+    quality_report.generate(real_data, synthetic_data, metadata.to_dict(), verbose)
+    return quality_report.get_score()
+
+
+def run_diagnostic(real_data, synthetic_data, metadata, verbose=True):
+    """Run diagnostic report for the synthetic data.
+
+    Args:
+        real_data (pd.DataFrame):
+            The table containing the real data.
+        synthetic_data (pd.DataFrame):
+            The table containing the synthetic data.
+        metadata (SingleTableMetadata):
+            The metadata object describing the real/synthetic data.
+        verbose (bool):
+            Whether or not to print report summary and progress.
+            Defaults to True.
+
+    Returns:
+        dict:
+            The diagonstic results.
+    """
+    quality_report = DiagnosticReport()
+    quality_report.generate(real_data, synthetic_data, metadata.to_dict(), verbose)
+    return quality_report.get_results()
```

Comment:
@fealho similarly, this should return the diagnostic_report object itself, not the score